src/main/java/org/folio/circulation/resources/LoanCollectionResource.java

```diff
@@ -128,7 +130,7 @@ void replace(RoutingContext routingContext) {
     final Clients clients = Clients.create(context, client);
     final RequestQueueRepository requestQueueRepository = RequestQueueRepository.using(clients);
     final ServicePointRepository servicePointRepository = new ServicePointRepository(clients);
-    final ItemRepository itemRepository = new ItemRepository(clients, false, false, false);
+    final ItemRepository itemRepository = new ItemRepository(clients, true, true, true);
```

Comment:
Can you explain why we need to fetch location, material type, and loan type?

Reply:
This information is needed for **patron notice template** context. We have the ability to insert **location**, **material type**, and **loan type** the information into patron notices.